apps/src/templates/teacherDashboard/RecentCourses.story.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
I'm realizing now that I could have a more generic collapsable section the is passed a heading, a link, and content. I'm going to end up making something very similar, for example, for the Manage Sections and Announcements & New sections. Opinions on whether I should generalize this component or make specific components for each section?